code/datums/traits/good.dm

```diff
@@ -29,6 +29,22 @@
 
 
 
+/datum/quirk/buster
+	name = "Buster"
+	desc = "Old fights have made you able to do more damage, although pretty inaccurately."
+	value = 3
+	gain_text = "<span class='notice'>You feel like you could cause more damage.</span>"
+	lose_text = "<span class='notice'>Your fists feel less dangerous.</span>"
+
+/datum/quirk/buster/add()
```

Comment:
no remove? doesn't update on species gain/loss?

Reply:
considering how I have to change this up, and it gets completely wiped on species change